docs/getting-started/running-rpm.md

```diff
@@ -21,9 +21,26 @@ to be installed:
 
 ```console
 sudo dnf install -y dnf-plugins-core
-sudo dnf copr enable @ebpf-sig/bpfman-next
 ```
 
+To install officially released versions:
+
+```console
+sudo dnf copr enable @ebpf-sig/bpfman
+```
+
+To install nightly builds:
+
+```console
+sudo dnf copr enable @ebpf-sig/bpfman
```

Comment:
Is this supposed to be @ebpf-sig/bpfman-next?

Reply:
Yep good catch